tmail-backend/integration-tests/jmap/jmap-integration-tests-common/src/main/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
what is it?

Reply:
configuration file for mock final `FirebaseMessagingException` class